website/docs/d/is_volumes.html.markdown

```diff
@@ -25,6 +25,7 @@ In addition to all argument references listed, you can access the following attr
 - `id` - The unique identifier of the VolumeCollection.
 - `volumes` - (List) Collection of volumes.
 	Nested scheme for **volumes**:
+	- `access_tags`  - (String) Access management tags associated for the volume.
```

Comment:
`access_tags` - (Optional, List of Strings)

Reply:
Changed to (List of Strings) as it is a computed field